tools/chatgpt/chatgpt.py

```diff
@@ -4,6 +4,8 @@
 from openai import OpenAI
 
 context_files = sys.argv[1].split(",")
+print(context_files)
+sys.exit(0)
```

Comment:
that is wired ... ;)

Reply:
It was just for testing I forgot to remove it, sorry!